plugins/modules/ec2_vol.py

```diff
@@ -385,21 +398,35 @@ def update_volume(module, ec2_conn, volume):
 
         target_throughput = module.params.get('throughput')
         throughput_changed = False
-        if 'gp3' in [target_type, original_type]:
-            if target_throughput:
+        if target_throughput is not None:
+            if 'gp3' in [target_type, original_type]:
                 original_throughput = volume.get('throughput')
                 if target_throughput != original_throughput:
                     throughput_changed = True
                     req_obj['Throughput'] = target_throughput
-
-        changed = iops_changed or size_changed or type_changed or throughput_changed
+            else:
+                module.warn("throughput parameter is valid only for gp3 volumes.")
+
+        target_multi_attach = module.params.get('multi_attach')
+        multi_attach_changed = False
+        if target_multi_attach is not None:
+            if "io1" in [target_type, original_type] or "io2" in [target_type, original_type]:
```

Comment:
aws console does not support it for io1. does boto3 really accept it for io1?

Reply:
according to documentation for both aws cli and boto3 api this is supported for io1 and io2 disks

https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-volumes-multi.html#enable
https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ec2.html#EC2.Client.create_volume